buildkite/scripts/deploy-nightly.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# TODO: make sure terraform installed
+
+# implicit flags needed to be passed in via runInDocker
+# BUILDKITE_GS_APPLICATION_CREDENTIALS_JSON
+# $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY
+
+source DOCKER_DEPLOY_ENV
+DAEMON_TAG="gcr.io/o1labs-192920/coda-daemon:${CODA_VERSION}-${CODA_GIT_HASH}"
+ARCHIVE_TAG="gcr.io/o1labs-192920/coda-archive:${CODA_VERSION}-${CODA_GIT_HASH}"
+
+echo "Nightly ENV: $NIGHTLY"
+echo "DAEMON_TAG: $DAEMON_TAG"
+echo "ARCHIVE_TAG: $ARCHIVE_TAG"
+
+if [ ! -z $NIGHTLY ]; then
+  echo "Deploying Nightly"
+
+  # Authenticate terraform for gcp
+  echo "$BUILDKITE_GS_APPLICATION_CREDENTIALS_JSON" > /tmp/gcp_creds.json
```

Comment:
Hey these can be removed if running from an agent. Probably best to just at most assert the envvars are set properly.